lib/fastly/acl_entry.rb

```diff
@@ -0,0 +1,28 @@
+class Fastly
+  class ACLEntry < Base
+    attr_accessor :service_id, :acl_id, :entry_id
```

Comment:
It looks like we are missing some required fields here:

* `ip`
* `subnet`
* `negated`
* `comment`

See the following for more details:
https://docs.fastly.com/api/config#acl_entry